packages/turf-helpers/index.js

```diff
@@ -248,6 +248,7 @@ function multiPolygon(coordinates, properties) {
  */
 function geometryCollection(geometries, properties) {
     if (!geometries) throw new Error('geometries is required');
```

Comment:
@DenisCarriere just being very fussy here, but from [specs](https://tools.ietf.org/html/rfc7946#section-3.1.8) geometryCollections can have an empty `geometries` array.
But `(!geometries)` would exclude that, right?

[Same goes](https://tools.ietf.org/html/rfc7946#section-3.3) actually for FeatureCollection, which I'm noticing has the same check.